survey/default.nix

```diff
@@ -634,12 +635,10 @@ let
     # So somehow, the above `zlib_static` uses *this* `zlib`, even though
     # the above uses `previous.zlib.override` and thus shouldn't see this one.
 
-    # The test-suite for PostgreSQL 13 fails:
-    # https://github.com/NixOS/nixpkgs/issues/150930
-    #
-    # Even if you disable the test-suite, there are various linking issues.
-    # PostgreSQL 12 has similar issues, so we drop to PostgreSQL 11.
-    postgresql = (previous.postgresql_11.overrideAttrs (old: { dontDisableStatic = true; })).override {
+    # Disable failing tests for postgresql on musl that should have no impact
+    # on the libpq that we need (collate.icu.utf8 and foreign regression tests)
+    # This approach is copied from PostgREST, see https://github.com/PostgREST/postgrest/pull/2002/files#diff-72929db01d3c689277a1e7777b5df1dbbb20c5de41d1502ff8ac6b443a4e74c6R45
+    postgresql = (previous.postgresql_14.overrideAttrs (old: { dontDisableStatic = true; doCheck = false; })).override {
```

Comment:
I copied this comment wholesale from PostgREST too.
I have no reason to doubt it (and have been running software in production with no issues), but I'm not an expert on this.